package.json

```diff
@@ -29,6 +29,10 @@
     "posteslint": "npm run prettier-readmes",
     "prettier-readmes": "prettier --write **/**/*.md --tab-width 3 --single-quote true",
     "start": "concurrently --kill-others --raw \"serve\" \"serve -p 5001 -c serve-test.json\" \"lerna run --ignore playground --parallel --stream start\"",
+    "start:docker": "npm run start:docker:build && npm run start:docker:up",
```

Comment:
What's your reasoning for providing these?

Reply:
Running Docker under WSL2 is slightly different when building and scaling out test agents. Devs will need to run 2 different scripts: `docker-compose -f docker-compose-wsl2.yml build --parallel` followed by `docker-compose -f docker-compose-wsl2.yml up --scale chrome=4`.

This `npm run start:docker` will run both scripts.